### ISSUE TITLE
[NFR] Post Scriptum

### ISSUE THREAD

**Issue by Alvares1998** [GameTactic/Issues#42]
# Basics

- [x] Needs maps to work.
- [x] Needs entities to work.
- [ ] I am officer in a clan/community.
- [ ] I am member of GameTactic Discord.
- [ ] I am willing to contribute (Provide resources like maps, entities and keep the up-to-date).

# Technical questions

- **Q:** Is there API available for entities and maps? 
    - **A:** YOUR_ANSWER_HERE
- **Q:** How often is entities and maps changing?
    - **A:** YOUR_ANSWER_HERE
- **Q:** Would you be able to get more interest for this issue?
    - **A:** YOUR_ANSWER_HERE

# Comments
  Would you add Post Scriptum to the available Games in Game Tactics? :)

#### Resources for maps and entity icons are available from following links:

**Comment by SaitoTachibana** [comment]
Agreed, we need this Post Scriptum badly, as we usually upload out Images and use it in the World of Tanks version :D

https://drive.google.com/drive/folders/1Bc0iNcgZP_wYzKQIuLAWTnkv3SwVCbCL
Files for most Main Game Maps. Additional Maps made by the Community may be requested ;D

**Comment by SaitoTachibana** [comment]
https://drive.google.com/drive/folders/1Bc0iNcgZP_wYzKQIuLAWTnkv3SwVCbCL
Files for most Main Game Maps. Additional Maps made by the Community may be requested ;D

**Comment by NikoGrano** [comment]
Hello @SaitoTachibana provide information how did you get these resources?

**Comment by Alvares1998** [comment]
Good Morning,

if you really think about adding Post Scriptum to Game Tactics, we would
create a drive folder with all ressources needed, including maps, icons
etc. + Attribution.

**Comment by NikoGrano** [comment]
We would need answers to updated issue description to continue.

**Comment by Alvares1998** [comment]
> We would need answers to updated issue description to continue.

I just read it and will take care of everything in the coming days!
Can you send an invitation link for the Game Tactics Discord in here, if it is available for public? I was searching for it yesterday, but didn't find anything.

**Comment by NikoGrano** [comment]
Can be found from our official site, wottactic.com.

https://discord.com/invite/9wqC34wK

**Comment by SaitoTachibana** [comment]
Hello @SaitoTachibana provide information how did you get these resources?
_Originally posted by @NikoGrano in https://github.com/GameTactic/Issues/issues/42#issuecomment-1034610254_

Official Post Scriptum Discord Community Resources. Made Public by Periscope Games

We would need answers to updated issue description to continue.
_Originally posted by @NikoGrano in https://github.com/GameTactic/Issues/issues/42#issuecomment-1034628874_

As far as the Technical questions go:

Q: Is there API available for entities and maps?
A: No, not to my knowledge. 

Q: How often are entities and maps changing?
A: Maps are Fixed. New Maps in form of Third-Party additions are possible. Changes are only made in form of addition of content. There are only vary rare additions aside the Third-Party content.

Q: Would you be able to get more interest for this issue?
A: The Post Scriptum Community is able to generate traction if needed for this issue. This tool as per is being widely used in different areas of Post Scriptum and as such, it is in the General interest of the Community of the Creation of a Postscriptumtactics.

**Comment by NikoGrano** [comment]
Developers of Post Scriptum have ignored all our messages, so for now we are not putting any priority on this issue. We are for now tweaking the current platform.

Tough, don't be sad, we will implement this later, but for now we do not have the resources and game developer ignoring us isn't making things any easier.

**Comment by SaitoTachibana** [comment]
Message from Community Manager Snazzyduckling regarding your previous Post:

Have you contacted us in the past? So sorry it hasn't been seen  Could you send an email with what you need from us to my own email hchristensen@periscope-games.com

I recommend going through the E-Mails are they are most responsive there.
